amoro-web/mock/modules/optimize.js

```diff
@@ -62,13 +71,14 @@ export default [
               }
             },
             "tableName": "test_catalog.db.user",
-            "optimizeStatus": "idle",
+            "status": "full_optimizing.commtting",
```

Comment:
Let's postpone this feature for now and keep this PR as clean and simple as possible.

Reply:
I get it.